ipyleaflet/basemaps.py

```diff
@@ -245,5 +247,20 @@
             min_zoom = 1,
             max_zoom = 18
         )
+    ),
+    HERE = Bunch(
```

Comment:
Maybe it would be a good idea to put those basemaps in a different bunch? I would say in `here.py`? So that there is a clear separation between basemaps without creds and basemaps with creds?

Reply:
Done. I still think a new subdir like `providers` or so would be a good idea, but I have a hard time to come up with an appropriate name. `providers` sounds too general while anything else I can think of would include `basemaps` and/or `credentials` or `proprietary` which sounds too long. Any idea?

Reply:
`external_providers`? Maybe something general is not too bad. I would like to add maki icons in this directory later so something containing `basemaps` in the name would not make sense in this case.

Reply:
So `external` it is now...

Reply:
Done.